README.flow-typed.txt

```diff
@@ -1,3 +1,15 @@
+# Making Flow Aware of the Types for Our Dependencies
```

Comment:
You have introduced Markdown markup to this file but it's still named ".txt", please rename to "README.flow-typed.md". And a super-minor update: we have been using a single space character following a period in a paragraph of text. Could you switch this file to use a single space, rather than 2?

Reply:
Done.

I haven't found anything yet, but it'd be nice to have light tooling (warnings in the IDE at least) to support conventions like that, as I have trouble remembering them otherwise.  We can revisit if it keeps coming up, especially if we start writing longer docs.